substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSections.java

```diff
@@ -351,7 +335,9 @@ public void installDebugInfo(DebugInfoProvider debugInfoProvider) {
             int lo = debugCodeInfo.addressLo();
             int hi = debugCodeInfo.addressHi();
             int primaryLine = debugCodeInfo.line();
+
             Range primaryRange = new Range(fileName, filePath, className, methodName, paramNames, returnTypeName, stringTable, lo, hi, primaryLine);
+            debugContext.log("PrimaryRange %s %s.%s", fileName, className, methodName);
```

Comment:
Using `.log(...)` like this will write logging messages with the default logging level `1`. This corresponds to using `-H:Log=<pattern>` without specifying the optional `[:<level>]` part. See https://github.com/oracle/graal/blob/master/compiler/src/org.graalvm.compiler.debug/src/org/graalvm/compiler/debug/doc-files/DumpHelp.txt